packages/flutter_tools/lib/src/bundle.dart

```diff
@@ -230,6 +230,6 @@ Future<void> writeBundle(
     assetEntries.entries.map<Future<void>>((MapEntry<String, DevFSContent> entry) async {
       final File file = fs.file(fs.path.join(bundleDir.path, entry.key));
       file.parent.createSync(recursive: true);
-      await file.writeAsBytes(entry.value.contentsAsBytes());
+      entry.value.copyToFile(file);
```

Comment:
It looks like the map closure no longer needs to be async.

Reply:
Done